test/e2e/registration_service_test.go

```diff
@@ -51,3 +56,415 @@ func (s *registrationServiceTestSuite) TestLandingPageReachable() {
 	require.NoError(s.T(), err)
 	assert.Equal(s.T(), http.StatusOK, resp.StatusCode)
 }
+
+func (s *registrationServiceTestSuite) TestHealth() {
+	s.Run("verify_healthcheck", func() {
+		// Call health endpoint.
+		req, err := http.NewRequest("GET", s.route+"/api/v1/health", nil)
+		require.NoError(s.T(), err)
+		client := &http.Client{
+			Timeout: time.Second * 10,
+			Transport: &http.Transport{
+				TLSClientConfig: &tls.Config{
+					InsecureSkipVerify: true,
+				},
+			},
+		}
+
+		resp, err := client.Do(req)
+		require.NoError(s.T(), err)
+		assert.Equal(s.T(), http.StatusOK, resp.StatusCode)
+
+		defer resp.Body.Close()
+
+		body, err := ioutil.ReadAll(resp.Body)
+		require.NoError(s.T(), err)
+		require.NotNil(s.T(), body)
+
+		mp := make(map[string]interface{})
+		err = json.Unmarshal([]byte(body), &mp)
+		require.Nil(s.T(), err)
+
+		// Verify JSON response.
+		alive := mp["alive"]
+		require.True(s.T(), alive.(bool))
```

Comment:
I think you should verify the type before casting to a `bool`

Reply:
(same for all other map entries)

Reply:
fixed here: 563467bf4c0dc459366a02e04e264965d15c61ca